cni/network/network_windows.go

```diff
@@ -267,13 +268,39 @@ func getPoliciesFromRuntimeCfg(nwCfg *cni.NetworkConfig) []policy.Policy {
 			Settings: rawPolicy,
 		})
 
-		policy := policy.Policy{
+		policyv4 := policy.Policy{
 			Type: policy.EndpointPolicy,
 			Data: hnsv2Policy,
 		}
-		log.Printf("[net] Creating port mapping policy: %+v", policy)
 
-		policies = append(policies, policy)
+		log.Printf("[net] Creating port mapping policy: %+v", policyv4)
+		policies = append(policies, policyv4)
+
+		// add port mapping policy for v6 if it's dualstack overlay mode
```

Comment:
I would suggest drop "if it's dualstack overlay mode". We want to set this for ipv6